### PR TITLE
Fix syntax error, lint target, and formatting.

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/102/
Test PASSed.

**Comment by bac** [comment]
The file hook/cached-website-relation-joined has a syntax error that was introduced before 2.1.1 was cut.  It is only showing up now in mojo CI since previous runs of CI installed v2.0.3 of the charm before upgrading to 2.1.1. The relation was created under 2.0.3 so this hook was not exercised. 

Now we install 2.1.1 from scratch and jenkins fails.

The error would've been caught if our lint target was better crafted.  That has been fixed now, which necessitated some formatting changes.


QA:

Deploy the charm and ensure things work.  Create a relation like:
`["jc-jujugui:cached-website", "jc-apache2:balancer"]`

**Comment by hatched** [comment]
:+1:

File: Makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ lint: setup
 	@$(VENV)/bin/flake8 --show-source \
 		--exclude=.venv,charmhelpers \
 		--filename *.py,20-functional.test \
-		hooks/ tests/ server/
+		hooks/* tests/ server/
```
Comment (frankban) [inline Makefile]: So this means lint is executed on all files, also non .py files?

File: hooks/cached-website-relation-joined [hunk 2]
```diff
@@ -17,7 +17,7 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-from charmhelpers.core.hookenv import
+from charmhelpers.core.hookenv import (
```
Comment (frankban) [inline hooks/cached-website-relation-joined]: Heh...

**Comment by jrwren** [comment]
:+1:

**Comment by frankban** [comment]
:+1:

File: Makefile (hunk 1 above)
Comment (bac) [inline Makefile]: Yes, which is what is required for full coverage.  Should we plop a bash script in there we'll be screwed.

**Comment by jrwren** [comment]
QA: bootstrap; deploy cs:apache2; deploy local:juju-gui; add-relation  "juju-gui:cached-website" "apache2:balancer"

There was no error. Relation was added.

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-charm-merge

**Pull request merged**